src/Microsoft.PowerShell.ConsoleHost/host/msh/UpdatesNotification.cs

```diff
@@ -62,7 +62,28 @@ internal static void ShowUpdateNotification(PSHostUserInterface hostUI)
                     ? ManagedEntranceStrings.StableUpdateNotificationMessage
                     : ManagedEntranceStrings.PreviewUpdateNotificationMessage;
 
-                string notificationMsg = string.Format(CultureInfo.CurrentCulture, notificationMsgTemplate, releaseTag);
+                string notificationColor = string.Empty;
+                string resetColor = string.Empty;
+
+                string line2Padding = string.Empty;
+                string line3Padding = string.Empty;
+
+                // We calculate how much whitespace we need to make it look nice
+                if (hostUI.SupportsVirtualTerminal)
+                {
+                    // Use Warning Color
+
+                    notificationColor = "\x1B[7m";
+                    resetColor = "\x1B[0m";
+
+                    // The first line is longest, if the message changes, this needs to be updated
+                    string[] lines = notificationMsgTemplate.Split('\n');
+                    line2Padding = line2Padding.PadRight(lines[0].Length - lines[1].Length + releaseTag.Length);
+                    line3Padding = line3Padding.PadRight(lines[0].Length - lines[2].Length + 3);  // 3 represents the extra placeholder
```

Comment:
Minor suggestion: how about using `IndexOf` to get the line length instead? that would avoid the allocation comes with `Split`

Reply:
BTW, is the URL `https://aka.ms/PowerShell-Release?tag=xxx` already working? It doesn't lead me to the Github release publish page.

Reply:
The URL is `https://aka.ms/PowerShell-Releases` (plural), because someone else took `https://aka.ms/PowerShell-Release` (singular).  I took ownership, so I'll update this PR to use the singular one which is already live.